WcaOnRails/app/models/registration.rb

```diff
@@ -86,6 +83,10 @@ def best_solve(event, type)
     person ? person.best_solve(event, type) : SolveTime.new(event.id, type, 0)
   end
 
+  def event_picker_events
+    registration_events.map(&:event_object).sort_by(&:rank)
+  end
```

Comment:
It would be nice to grep for occurrences of this method calls. One I remember is [here](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/views/shared/_associated_events_picker.html.erb#L27), it would be sufficient to call `f.object.event` (`f.object` is one of the associated events, for example an RegistrationEvent).